scenarioo-client/app/build/mainpage/mainpage.component.css

```diff
@@ -2,3 +2,10 @@
     border: 0;
     margin-bottom: 20px;
 }
+
+#grayBackground {
```

Comment:
Maybe _#tabBackground_ instead? I think _grayBackground_ is a bit ambiguous, like it is used only to set a gray background. But it contains additional stylings specific to the tab background.

Reply:
@lynparbone yeah, this was an old naming, I also thought it is a bad name, but was not so sure what would be a better name.
Maybe "tabHeader" because indead it is before the tabs, and not really the tabs?

Reply:
Yes, I think _tabHeader_ would be suitable as well!